app/src/processing/app/Base.java

```diff
@@ -1481,19 +1481,22 @@ public void actionPerformed(ActionEvent actionevent) {
     ButtonGroup boardsButtonGroup = new ButtonGroup();
     Map<String, ButtonGroup> buttonGroupsMap = new HashMap<>();
 
-    List<JMenu> platformMenus = new ArrayList<JMenu>();
+    class PlatformJMenu extends JMenu {
+      public PlatformJMenu(TargetPlatform pl) {
+        String text = pl.getPreferences().get("name");
+        if (text == null) {
+          text = pl.getContainerPackage().getId() + "-" + pl.getId();
+        }
+        setText(text);
+      }
+    }
```

Comment:
Now that untranslated strings are no longer needed, all this code seems no longer needed? This commit could just be a matter of removing `tr()`, no point in creating the `PlatformJMenu` class anymore I think?

Reply:
done